### PR TITLE
feat: upgrade no breaking modules

### PR DESCRIPTION
**Pull request by gusliedke**
Updating a few module

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -81,7 +81,7 @@
     "eslint-plugin-react": "^7.24.0",
     "eslint-plugin-react-hooks": "^4.0.5",
     "husky": "^7.0.2",
-    "jest": "^27.0.6",
+    "jest": "^26.1.0",
```
Comment (AndyEPhipps) [inline package.json:84]: What about Jest was breaking? Isn't it worth trying to address now because we'll have to update it eventually anyways?

Comment (gusliedke) [reply package.json:84]: It was an issue with `react-test-renderer` .. it got a bit messy with all tests and snaps updates I had to do and still couldn't get all errors sorted. I'd prefer to tackle it in a separated PR and leave this just with the Dependabot updates.

Comment (AndyEPhipps) [reply package.json:84]: Understood!

**Review by AndyEPhipps — APPROVED** [review]

**Pull request merged**